tests/utils/test_String.cpp

```diff
@@ -14,7 +14,7 @@
 UTEST(test_String, DefaultConstruct)
 {
     // The string should default construct correctly
-    String s;
+    Siege::String s;
```

Comment:
I think in the String test file we can just have a `using` directive at the top.

Reply:
I honestly don't think it makes that big of a difference but I went ahead and added the change.